manifests/FineTuning/xeon/finetuning.yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+   name: opea-finetune
+   namespace: default
+spec:
+   replicas: 1
+   selector:
+      matchLabels:
+         service: opea-finetune
+   template:
+      metadata:
+         labels:
+            service: opea-finetune
+      spec:
+         containers:
+            - name: opea-finetune
+              image: opea-finetune:latest
+              imagePullPolicy: IfNotPresent
+---
+apiVersion: v1
+kind: Service
+metadata:
+   name: opea-finetune-entrypoint
+   namespace: default
+spec:
+   type: NodePort
```

Comment:
If NodePort is not must, please remove it. NodePort will cause conflict in the CI test machine.

Reply:
thanks, I will remove that. do you have a port assignment rule for each new microservice?

Reply:
not much rules. It's better to use a same port as mega service but using ClusterIP to avoid hard coded port number. For your reference: https://github.com/opea-project/GenAIInfra/blob/main/manifests/DocSum/xeon/tgi_service.yaml#L17